\hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1TimeoutError}{}\doxysection{pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Timeout\+Error Class Reference}
\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1TimeoutError}\index{pip.\_vendor.urllib3.exceptions.TimeoutError@{pip.\_vendor.urllib3.exceptions.TimeoutError}}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Raised when a socket timeout error occurs.

Catching this error will catch both :exc:`ReadTimeoutErrors
<ReadTimeoutError>` and :exc:`ConnectTimeoutErrors <ConnectTimeoutError>`.
\end{DoxyVerb}
 Inheritance diagram for pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Timeout\+Error\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=4.444445cm]{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1TimeoutError}
\end{center}
\end{figure}


The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/\+Lib/site-\/packages/pip/\+\_\+vendor/urllib3/exceptions.\+py\end{DoxyCompactItemize}
